frontend/src/plugin/registry.tsx

```diff
@@ -194,6 +195,10 @@ export default class Registry {
     store.dispatch(setDetailsViewHeaderAction(actionFunc));
   }
 
+  registerKubeObjectGlance(glance: Glance) {
```

Comment:
adding a register method here is not required, it's deprecated. Having a register function that you've added on line 305 is enough